triggers/createFolder.js

```diff
@@ -0,0 +1,198 @@
+var hookID = "";
+const eventIds = [];
+
+const subscribeHook = async (z, bundle) => {
```

Comment:
combine the code for createFolder, createFiler, and deleteFile

Reply:
Fetch hookID from bundle.subscribeData!.id
refer postHog :- 

https://github.com/PostHog/posthog-zapier/blob/main/src/utils.ts

Reply:
resolved